src/components/HospitalCardDetails.tsx

```diff
@@ -199,50 +228,70 @@ export const HospitalCardDetails: React.FC<{ hospital: Hospital }> = ({
                 </ActionButton>
               </Stack>
             </CardContent>
-            <CardContent sx={{ flex: 1 }}>
+            <CardContent
+              sx={{
+                flex: 1,
+                display: "flex",
+                flexDirection: "column",
+                gap: 0.8,
+              }}
+            >
+              <Box
+                display="flex"
+                alignItems="center"
+                marginTop={1}
+                sx={{
+                  backgroundColor: "#FFFCD8",
+                  borderRadius: "8px",
+                  padding: "2px 10px 2px 10px",
+                  width: "245px",
+                }}
+              >
+                <Typography
+                  variant="body2"
+                  color="textSecondary"
+                  marginRight={1}
+                  sx={{ color: "#0000000" }}
+                >
+                  Matched by {partnerName}
+                </Typography>
+              </Box>
               <Typography variant="body2" color="textSecondary">
-                <span style={{ color: "black" }}>25k </span> raised of 100k -{" "}
+                <span style={{ color: "#0000000" }}>
+                  ${hospital.matchedFunded?.fundingCompleted || 0}{" "}
+                </span>{" "}
+                <span style={{ color: "#828282" }}>
+                  raised of ${hospital.matchedRequest?.requested || 0} -{" "}
+                </span>
                 <Typography
                   variant="body2"
                   component="span"
                   color="success.main"
-                  sx={{ fontStyle: "italic", color: "#92c65e" }}
+                  sx={{
+                    fontStyle: "italic",
+                    color: hospital.status === "active" ? "#92c65e" : "#DB5757",
+                  }}
                 >
                   {hospital?.status}
                 </Typography>
               </Typography>
 
               <Typography variant="body2" color="textSecondary">
-                400+ kids impacted
+                {hospital.year}+ kids impacted
               </Typography>
 
-              <Box display="flex" alignItems="center" marginTop={1}>
-                <Typography
-                  variant="body2"
-                  color="textSecondary"
-                  marginRight={1}
-                >
-                  Matched by
-                </Typography>
-                <Avatar
-                  alt="Organization Logo"
-                  src="/path/to/profile1.jpg"
-                  sx={{ width: 20, height: 20, marginLeft: 1 }}
-                />
-                <Avatar
-                  alt="Organization Logo"
-                  src="/path/to/profile2.jpg"
-                  sx={{ width: 20, height: 20, marginLeft: 1 }}
-                />
-                +
-              </Box>
               <Typography
```

Comment:
We should create an "Italicized" style for this.  It is used in multiple places.